lib/order.rb

```diff
@@ -0,0 +1,68 @@
+require_relative 'menu'
+require_relative 'confirmation'
+require 'dotenv'
+Dotenv.load
+
+class Order
+
+  attr_reader :selection, :menu, :prices, :total, :orders, :confirmation
+
+  def initialize(menu = Menu.new)
+  @menu = menu
+  @orders = []
+  @selection = {}
+  @prices = []
+  end
+
+  def start
+    puts @menu.introduction
+    puts @menu.show_menu
+  end
+
+  def select_order(item, amount)
+    @selection = { item.to_sym => amount}
+    add_to_orders
+    return "#{amount} x #{item} added to your order!"
```

Comment:
I know that the Ruby style would generally only use an explicit returns for an early return out of the method, e.g. if the method has two conditions and it meets the first. Ruby is more focused on code minimalism.